README.md

```diff
@@ -92,6 +92,8 @@ module "lambda" {
 
       // optionally overwrite `cloudwatch_event_target_arn` in case an alias should be used for the event rule
       cloudwatch_event_target_arn = aws_lambda_alias.example.arn
+      // optionally add `cloudwatch_event_target_input` for event input
```

Comment:
the README.md is generated, could you please add this to `docs/part1.md` and run `make documentation`

Please also add this to the example in https://github.com/moritzzimmer/terraform-aws-lambda/blob/f195d88aa57f4fe08d5cdeede5d173cf26ee0961/examples/with-cloudwatch-event-rules/main.tf#L29